From 39539d7b882722336bb4bfad99ef3ebadfc9b276 Mon Sep 17 00:00:00 2001
From: amitkumar50 <amitkuma@redhat.com>
Date: Tue, 10 Apr 2018 15:29:01 +0530
Subject: [PATCH] MAN: Add sss-certmap man page regarding priority processing

PR adds following text in PRIORITY section of man sss-certmap:
The processing is stopped when a matched rule is found and no
further rules are checked.

Resolves: https://pagure.io/SSSD/sssd/issue/3469

Reviewed-by: Justin Stephenson <jstephen@redhat.com>
(cherry picked from commit 56839605d139573319b7df24774b56ea78ec742b)
---
 src/man/sss-certmap.5.xml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/man/sss-certmap.5.xml b/src/man/sss-certmap.5.xml
index 593cd4666..db258d14a 100644
--- a/src/man/sss-certmap.5.xml
+++ b/src/man/sss-certmap.5.xml
@@ -44,7 +44,9 @@
         <para>
             The rules are processed by priority while the number '0' (zero)
             indicates the highest priority. The higher the number the lower is
-            the priority. A missing value indicates the lowest priority.
+            the priority. A missing value indicates the lowest priority. The
+            rules processing is stopped when a matched rule is found and no
+            further rules are checked.
         </para>
         <para>
             Internally the priority is treated as unsigned 32bit integer, using
-- 
2.14.3

